GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -297,13 +297,14 @@ public static HotkeySettings[] CreateDefaultSettings()
                     Hk(RevisionGridControl.Command.CompareToCurrentBranch, Keys.None),
                     Hk(RevisionGridControl.Command.CompareToWorkingDirectory, Keys.Control | Keys.D),
                     Hk(RevisionGridControl.Command.CreateFixupCommit, Keys.Control | Keys.X),
-                    Hk(RevisionGridControl.Command.DisableRevisionFilter, Keys.Control | Keys.Shift | Keys.F),
                     Hk(RevisionGridControl.Command.GoToChild, Keys.Control | Keys.N),
                     Hk(RevisionGridControl.Command.GoToCommit, Keys.Control | Keys.Shift | Keys.G),
                     Hk(RevisionGridControl.Command.GoToParent, Keys.Control | Keys.P),
                     Hk(RevisionGridControl.Command.NextQuickSearch, Keys.Alt | Keys.Down),
                     Hk(RevisionGridControl.Command.OpenCommitsWithDifftool, Keys.None),
                     Hk(RevisionGridControl.Command.PrevQuickSearch, Keys.Alt | Keys.Up),
+                    Hk(RevisionGridControl.Command.ResetPathRevisionFilter, Keys.Control | Keys.Shift | Keys.I),
+                    Hk(RevisionGridControl.Command.ResetRevisionFilter, Keys.Control | Keys.I),
                     Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.F),
```

Comment:
With #9827 it is more important that shortcuts are unique in all forms.
The existing `ctrl-f` should probably be changed, which affect the new shortcuts too.
@mstv any better suggestions?
```suggestion
                    Hk(RevisionGridControl.Command.ResetPathRevisionFilter, Keys.Control | Keys.Shift | Keys.H),
                    Hk(RevisionGridControl.Command.ResetRevisionFilter, Keys.Control | Keys.I),
                    Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.I),
```

Reply:
Hotkeys _can_ have different behavior depending on the focused control. You are right, if we can avoid it, it will save some manual focus switches.

```suggestion
                    Hk(RevisionGridControl.Command.ResetPathRevisionFilter, Keys.Control | Keys.Shift | Keys.H),
                    Hk(RevisionGridControl.Command.ResetRevisionFilter, Keys.Control | Keys.Shift | Keys.I),
                    Hk(RevisionGridControl.Command.RevisionFilter, Keys.Control | Keys.I),
```
was meant. I like the pair `Ctrl+I` / `Ctrl+Shift+I`.

Reply:
> Hotkeys _can_ have different behavior depending on the focused control. 
> You are right, if we can avoid it, it will save some manual focus switches.

There will be different actions for different controls. like Ctrl-A has a meaning in the FileViewer and FileStatusList, but we should avoid it so you do not have to focus the right control .

> was meant. I like the pair `Ctrl+I` / `Ctrl+Shift+I`.

Yes...